example/app.js

```diff
@@ -14,6 +25,144 @@ function test() {
     console.log(checks);
 }
 
+/**
+*Custom health check using our own class that extends Check
+*/
+function setUpCustomClassCheck() {
+    let config = {
+        name: 'CUSTOM CLASS Check for CNN Homepage',
+        severity: 2,
+        businessImpact: 'Its a HUGE deal',
+        technicalSummary: 'god knows',
+        panicGuide: 'Don\'t Panic',
+        interval: '10s'
+    };
+
+    theCheck = health.runCustomCheckClass(customCheck, config);
+    //TODO  modify health.runCustomCheck to supports running multiple custom check classes
+
+}
+
+/**
+*Configures a custom health checks using a config object
+*/
+function customHealthCheckConfig() {
+    let config = {
+        name: 'Custom Config Checks App Called Test',
+        description: 'json check fixture description',
+        checks: [
+            {
+                type: 'custom',
+                name: 'Custom Check some dependency 1 for App Called Test',
+                url: 'http://www.cnn.com/_healthcheck',
+                severity: 2,
+                businessImpact: 'Its a HUGE deal',
+                technicalSummary: 'god knows',
+                panicGuide: 'Don\'t Panic',
+                checkResult: {
+                    PASSED: 'Text if check passed',
+                    FAILED: 'Text is check failed',
+                    PENDING: 'This check has not yet run'
+                },
+                interval: '10s',
+                callback: function (json) {
+                    return json.version;
+                },
+                tick: function () {
+                    return fetch('http://www.cnn.com/_healthcheck')
+                        .then((response) => {
+                            let message;
+
+                            if (!response.ok) {
+                                message = util.format('BadResponse %s', response.status);
```

Comment:
No need to use util anymore. You can use a tick comment like so `BadResponse ${response.status}`